### ISSUE TITLE
Testing on Tanks and Temples

### ISSUE THREAD

**Issue by 05063112lcs** [TQTQliu/ET-MVSNet#14]
Hello, thank you very much for your excellent work, I would like to know, for the TNT dataset, how do you get your own pre-trained model, or is it just training on the DTU dataset and testing on the rest of the dataset.

**Comment by TQTQliu** [comment]
Hi, thanks for your interest. Before evaluating on Tanks and Temples (TNT) benchmark, we finetune our DTU trained model on the BlendedMVS dataset for the adaptation to the real-world scenes. This is a common practice in the MVS field. The command can be found [here](https://github.com/TQTQliu/ET-MVSNet/blob/e29d0b6c1d7772a86b730f4b8b106df3846a7fb2/README.md?plain=1#L91).

**Comment by 05063112lcs** [comment]
Thank you very much for your reply. Do you mean that the pre-trained model of the TNT dataset can be obtained through the training script of BlendMVS?



---Original---
From: "Tianqi ***@***.***&gt;
Date: Thu, Jun 6, 2024 23:17 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] Testing on Tanks and Temples (Issue #14)




 
Hi, thanks for your interest. Before evaluating on Tanks and Temples (TNT) benchmark, we finetune our DTU trained model on the BlendedMVS dataset for the adaptation to the real-world scenes. This is a common practice in the MVS field. The command can be found here.

**Comment by TQTQliu** [comment]
Yes, just set `BLD_CKPT_FILE` to the DTU trianed model and run the BlendedMVS training script.

**Comment by 05063112lcs** [comment]
I understand that for your article, you trained on DTU data set, tested on DTU data set,BlendedMVS and TNT data set, and then obtained the effect in the chart in the paper. Well, I'm replicating your work



---Original---
From: "Tianqi ***@***.***&gt;
Date: Thu, Jun 6, 2024 23:31 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] Testing on Tanks and Temples (Issue #14)




 
Yes, just set BLD_CKPT_FILE to the DTU trianed model and run the BlendedMVS training script.

**Comment by TQTQliu** [comment]
Thanks! 
To be precise, we train model on the dtu training set from scratch and then test it on the dtu test set. 
We then fine-tune the trained model on the blendedmvs dataset and test it on TNT.

**Comment by 05063112lcs** [comment]
Thank you for your trouble. Good luck with your studies



---Original---
From: "Tianqi ***@***.***&gt;
Date: Thu, Jun 6, 2024 23:47 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [TQTQliu/ET-MVSNet] Testing on Tanks and Temples (Issue #14)




 
Thanks!
 To be precise, we train model on the dtu training set from scratch and then test it on the dtu test set.
 We then fine-tune the trained model on the blendedmvs dataset and test it on TNT.
